addons/mail/static/src/components/message/message.js

```diff
@@ -546,6 +546,15 @@ class Message extends Component {
         this.message.originThread.open();
     }
 
+    /**
+     * @private
+     */
+    _onClickQueueError(ev) {
+        ev.preventDefault();
+        this.message.update({ hasSendError: false });
+        this.message.composer.processMessagesToBeSent();
+    }
+
```

Comment:
As said in another comment, would rename handler so it's less technical and more meaningful to actual user interaction (I think `_onClickSendAgain`).